src/server.js

```diff
@@ -103,29 +102,39 @@ async function startServer (options) {
   }
 
   server.makeOp = async function (bot) {
-    wrap.writeServer(`op ${bot.username}`)
+    wrap.writeServer(`op ${bot.username}\n`)
     await awaitServerMessage(bot, 'chat.type.admin')
+
+    console.log(`Make ${bot.username} OP.`)
   }
 
   server.teleport = async function (bot, pos) {
-    wrap.writeServer(`tp ${bot.username} ${pos.x} ${pos.y} ${pos.z}`)
-    await bot.waitForChunksToLoad()
+    wrap.writeServer(`tp ${bot.username} ${pos.x} ${pos.y} ${pos.z}\n`)
+    await once(bot, 'forcedMove')
```

Comment:
Oh, I meant to wait for forcedMove and then wait for chunks, but yes.

Reply:
done